website/docs/concepts/router.mdx

```diff
@@ -110,18 +112,12 @@ enum Route {
     Home,
     #[at("/post/:id")]
     Post { id: String },
-    // ...
 }
-```
-
-You can then access the post's id inside `<Switch />` and forward it to the appropriate component via properties.
 
-```rust ,ignore
-fn switch(routes: &Route) -> Html {
-    match routes {
+fn switch(route: &Route) -> Html {
+    match route {
         Route::Home => html! { <h1>{ "Home" }</h1> },
-        Route::Post { id } => <Post {id} />,
```

Comment:
this example wasn't working and was much cringe. I made it work and removed the ignore.